osu.Game.Tests/Chat/MessageFormatterTests.cs

```diff
@@ -273,6 +273,66 @@ public void TestMarkdownFormatLinkWithEscapedAndBalancedBrackets()
             Assert.AreEqual(21, result.Links[0].Length);
         }
 
+        [Test]
+        public void TestMarkdownFormatLinkWithInlineTitle()
+        {
+            Message result = MessageFormatter.FormatMessage(new Message { Content = "I haven't seen [this link format](https://osu.ppy.sh \"osu!\") before..." });
```

Comment:
//Arragne 
var blub = "someStuff";
var expected = "SomeStuff";

//Act 
var result = doSomeStuff(blub)

//Assert 
Assert.Something(expected, result);
 ?


have a detailed look [here](https://docs.microsoft.com/en-us/visualstudio/test/unit-test-basics?view=vs-2019)

it would make the test more readable and mantainable

Reply:
I am familiar with the arrange-act-assert pattern (also known as the given-when-then pattern). I use it in my own projects. If you notice these tests also are arranged similarly, but don't have the comment lines - and I don't believe adding them is so important as to break existing conventions in this repository.,